devices/usbc-cdc/usbphy-imxrt117x.c

```diff
@@ -116,8 +116,9 @@ void phy_init(void)
 
 	phyusb_common.usedPools = 0;
 
-	/* Enable USB clock gate */
-	_imxrt_setDirectLPCG(pctl_lpcg_usb, 1);
+	/* Enable USB clock gate, that clock is needed in RUN level only */
+	_imxrt_setDirectLPCG(pctl_lpcg_usb, clk_state_run);
+	_imxrt_setLevelLPCG(pctl_lpcg_usb, clk_level_run);
```

Comment:
Why not in all levels?

Reply:
I just followed the original boot loader approach.  They set `level=1 (run)` at the initialization of hid-device.

Reply:
Doesn't really matter as we don't use any lp modes (especially in plo)